test/support/factory.ex

```diff
@@ -24,6 +24,31 @@ if Code.ensure_loaded?(ExMachina) and Code.ensure_loaded?(Faker) do
       Trip
     }
 
+    def plan_factory do
+      %Plan{
+        date: Faker.DateTime.forward(2),
+        itineraries: __MODULE__.build_list(3, :itinerary),
+        routing_errors: Faker.Util.pick([__MODULE__.build_list(1, :routing_error), []]),
```

Comment:
I would split out a `plan_with_errors` factory from a `plan_factory`. I would be surprised if I went `build(:plan)` and got a plan that had errors.